validators/src/unit-test/java/uk/gov/moj/sdt/validators/BulkSubmissionValidatorTest.java

```diff
@@ -62,67 +59,81 @@
 import uk.gov.moj.sdt.domain.api.IIndividualRequest;
 import uk.gov.moj.sdt.domain.cache.api.ICacheable;
 import uk.gov.moj.sdt.utils.Utilities;
-import uk.gov.moj.sdt.utils.concurrent.api.IInFlightMessage;
 import uk.gov.moj.sdt.validators.exception.CustomerNotSetupException;
 import uk.gov.moj.sdt.validators.exception.CustomerReferenceNotUniqueException;
 import uk.gov.moj.sdt.validators.exception.RequestCountMismatchException;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
 /**
  * Tests for {@link BulkSubmissionValidatorTest}.
  *
  * @author d120520
  */
-public class BulkSubmissionValidatorTest extends AbstractValidatorUnitTest {
+@ExtendWith(MockitoExtension.class)
+class BulkSubmissionValidatorTest extends AbstractValidatorUnitTest {
+
+    private static final String ERROR_CODE_INCORRECT = "Error code incorrect";
+
+    private static final String SUBSTITUTION_VALUE_INCORRECT = "Substitution value incorrect";
+
+    private static final Long LONG_12345 = 12345L;
```

Comment:
Would be better to have a more descriptive name for this constant.  The value is an SDT customer id so name it along those lines.

Reply:
Constant has been renamed to SDT_CUSTOMER_ID.